### PR TITLE
fixes for proposal and lock record fixes

### PR DESCRIPTION
**Pull request by sk-enya**
:clipboard: Add associated issues, tickets, docs URL here.

## Overview
Fixes for annoying console errors from fetchProposal, Lock Records
## Changes

- Describe your changes and implementation choices. More details make PRs easier to review.
- Fixes for annoying console errors from fetchProposal, Lock Records
- Imported the beamer image in ecosystem.

## Testing

Open gateway and check consoles.

### CODE REVIEW

File: packages/boba/gateway/src/services/networkService.js [hunk 1]
```diff
@@ -4829,7 +4829,13 @@ class NetworkService {
       return
     }
 
+    if (!+process.env.REACT_APP_ENABLE_LOCK_PAGE) {
```
Comment (boyuan-chen) [inline packages/boba/gateway/src/services/networkService.js:4825]: Is it a new environment variable?

**Review by boyuan-chen — APPROVED** [review]

File: packages/boba/gateway/src/services/networkService.js [hunk 2]
```diff
@@ -4152,17 +4150,20 @@ class NetworkService {
...
         let endTimestamp = proposalData.endTimestamp.toString()
 
         let proposal = await delegateCheck.getActions(i+2)
 
         let hasVoted = null
 
-        if( this.account ) {
-          hasVoted = await delegateCheck.getReceipt(proposalID, this.account)
-        }
+        // NOTE: as per the veBoba getReciept expecting params as proposalId & tokenId so this will break
```
Comment (souradeep-das) [inline packages/boba/gateway/src/services/networkService.js]: should we remove this then?

**Review by souradeep-das — APPROVED** [review]

File: packages/boba/gateway/src/services/networkService.js (hunk 1 above)
Comment (sk-enya) [reply packages/boba/gateway/src/services/networkService.js:4825]: yeah as we have Lock Page so to disable that we need to have this.

Default value should be 0.

Whenever we have to enable the lock page we will remove this param.

File: packages/boba/gateway/src/services/networkService.js (hunk 2 above)
Comment (sk-enya) [reply packages/boba/gateway/src/services/networkService.js]: Yeah, okay we can remove this.

**Pull request merged**